tests/test_neighbors.py

```diff
@@ -241,3 +242,26 @@ def test_restore_n_neighbors(neigh, conv):
         ad.uns["neighbors"] = dict(connectivities=conv(neigh.connectivities))
     neigh_restored = Neighbors(ad)
     assert neigh_restored.n_neighbors == 1
+
+
+def test_neighbors_distance_equivalence():
```

Comment:
please parametrize this test over `n_neighbors`, `method`, and `key_added` (2 options each) to make sure they all get passed down correctly